### PR TITLE
Update UC docs

### PR DESCRIPTION
**Pull request by dennyglee**
**PR Checklist**

- [X] A description of the changes is added to the description of this PR.
- [ ] If there is a related issue, make sure it is linked to this PR.
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added or modified a feature, documentation in `docs` is updated

**Description of changes**
- Fixed various formatting and configuration settings for the docs site.  
- Added Spark integration
- Added Model integration
- Includes coded tabs, admonitions, linked tabs, and other docs improvements.

### CODE REVIEW

File: docs/rapidstart.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Rapidstart \- Hello UC\!
+# Rapidstart
 
 This document provides an end-to-end installation and basic testing of all current components of Unity Catalog. It allows developers to try out the major aspects of Unity Catalog on their local machine. This rapid start will also be the basis for the full Unity Catalog Docker.
 
-# Quick Start
+## Quick Start
```
Comment (tdas) [inline docs/rapidstart.md:5]: quick start inside rapid start?

Comment (dennyglee) [reply docs/rapidstart.md:5]: We will be removing this rapidstart after adding the UI and auth pages.

File: docs/usage/models.md [hunk 2]
```diff
@@ -1,49 +1,209 @@
...
-![UC Functions](../assets/images/uc_functions.png)
+![Unity Catalog and MLflow](../assets/images/unitycatalog-and-mlflow.png)
 
-Let's look at how this works.
+The MLflow client connects both MLflow and Unity Catalog services (via `port:5000` and `port:8080` respectively).  
 
-## Set Up
+1. When you are running your training runs with MLflow, the metrics generated in those training runs are stored within MLflow.
+2. For those same training runs, models are often generated, and those models are also stored within MLflow.
+3. Once you have decided which model you want to register, you can register and store that model into Unity Catalog
```
Comment (tdas) [inline docs/usage/models.md:18]:
```suggestion
3. Once you have decided which model you want to register, you can register and store that model into Unity Catalog.
```

File: docs/usage/models.md [hunk 3]
```diff
@@ -1,49 +1,209 @@
...
+```bash
+Successfully registered model 'unity.default.iris'.
+2024/09/24 20:51:29 INFO mlflow.store.model_registry.abstract_store: Waiting up to 300 seconds for model version to finish creation. Model name: unity.default.iris, version 1
+Created version '1' of model 'unity.default.iris'.
+<mlflow.models.model.ModelInfo object at 0x13ffb2e80>
+2024/09/24 20:51:29 INFO mlflow.tracking._tracking_service.client: 🏃 View run glamorous-gnu-508 at: http://127.0.0.1:5000/#/experiments/0/runs/b3ab03114a4b4b869f3d30070ffc8a1f.
+2024/09/24 20:51:29 INFO mlflow.tracking._tracking_service.client: 🧪 View experiment at: http://127.0.0.1:5000/#/experiments/0.
+```
+
+The results can be seen in the Unity Catalog UI at [http://localhost:3000,](http://localhost:3000) per the instructions in the [Interact with the Unity Catalog tutorial](https://github.com/unitycatalog/unitycatalog?tab=readme-ov-file#interact-with-the-unity-catalog-ui).
```
Comment (tdas) [inline docs/usage/models.md:124]:
```suggestion
The results can be seen in the Unity Catalog UI at [http://localhost:3000](http://localhost:3000) as per the instructions in the [Interact with the Unity Catalog tutorial](https://github.com/unitycatalog/unitycatalog?tab=readme-ov-file#interact-with-the-unity-catalog-ui).  
```

**Comment by dennyglee** [comment]
Thanks for the review @tdas

File: docs/usage/models.md [hunk 4]
```diff
@@ -1,49 +1,209 @@
...
+2024/09/24 20:51:29 INFO mlflow.store.model_registry.abstract_store: Waiting up to 300 seconds for model version to finish creation. Model name: unity.default.iris, version 1
+Created version '1' of model 'unity.default.iris'.
+<mlflow.models.model.ModelInfo object at 0x13ffb2e80>
+2024/09/24 20:51:29 INFO mlflow.tracking._tracking_service.client: 🏃 View run glamorous-gnu-508 at: http://127.0.0.1:5000/#/experiments/0/runs/b3ab03114a4b4b869f3d30070ffc8a1f.
+2024/09/24 20:51:29 INFO mlflow.tracking._tracking_service.client: 🧪 View experiment at: http://127.0.0.1:5000/#/experiments/0.
+```
+
+The results can be seen in the Unity Catalog UI at [http://localhost:3000](http://localhost:3000) as per the instructions in the [Interact with the Unity Catalog tutorial](https://github.com/unitycatalog/unitycatalog?tab=readme-ov-file#interact-with-the-unity-catalog-ui).  
+
+<!-- Change the above link to point to the UI section of the docs -->
```
Comment (tdas) [inline docs/usage/models.md:126]: Note this. it can be done right now.

**Review by tdas — APPROVED** [review]

**Pull request merged**